app/views/shared/_youtube_api.slim

```diff
@@ -0,0 +1,6 @@
+- if [@run, *@compare_runs].compact.map(&:video).compact.map { |video| video.youtube? }.any?
```

Comment:
Do you know if this will have any negative effects if it's run more than once before a page reloads?  Turbolinks doesn't clear the page completely, just replaces the body and merges the head so there's a chance for it to download the API stuff more than once if a user navigates to a page with a YT video multiple times in one session.

Reply:
Just thinking it through, I feel like there could be weirdness with Turbolinks. If we were to load the API JS more than once, I don't think the end user would be negatively impacted, but it feels like there might be a potential situation where the API JS *doesn't* load like expected.

I'll do some searching to see if people have mentioned specific interactions between Turbolinks and the YouTube IFrame API JS.